pkg/agent/multicast/mcast_controller.go

```diff
@@ -265,6 +266,10 @@ type Controller struct {
 	installedNodes      sets.Set[string]
 	encapEnabled        bool
 	flexibleIPAMEnabled bool
+	// ipv4Enabled is the flag that if it is running on IPv4 cluster. An error is returned if IPv4Enabled is false
+	// in Initialize as Multicast does not support IPv6 for now.
+	// TODO: remove this flag after IPv6 is supported in Multicast.
+	ipv4Enabled bool
```

Comment:
I feel like since we are adding this flag we should also add `ipv6Enabled`. This way we can log a message if `ipv4Enabled && ipv6Enabled` to inform the user that Multicast is only supported for IPv4 (but no need to hard fail in this case)

Reply:
sure, I will update.